workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Jobs/CodegenJob.cs

```diff
@@ -15,15 +15,20 @@ public class IgnoreCodegenJobAttribute : Attribute
 
     public abstract class CodegenJob
     {
-        public IEnumerable<string> OutputFiles => outputFiles;
+        public IEnumerable<string> ExpectedOutputFiles
+            => expectedOutputFiles
+                .Select(filePath => Path.Combine(OutputDirectory, filePath))
+                .Union(jobTargets.Select(target => target.FilePath));
```

Comment:
Why is `expectedOutputFiles` not already _all_ the expected output files?

Reply:
because this information is all inside the jobtargets in the new system

though one could argue we can just add to it each time a job comes in --> consider in cleanup PR

Reply:
Is it in the cleanup PR?

Reply:
![image](https://user-images.githubusercontent.com/10051819/74669870-f1680f00-519f-11ea-9155-358662905db8.png)